embassy-stm32/src/rcc/u5.rs

```diff
@@ -45,6 +45,18 @@ pub struct PllConfig {
     /// The multiplied clock – `source` divided by `m` times `n` – must be between 128 and 544
     /// MHz. The upper limit may be lower depending on the `Config { voltage_range }`.
     pub n: Plln,
+    /// The divider for the P output.
+    ///
+    /// When used to drive the system clock, `source` divided by `m` times `n` divided by `r`
```

Comment:
this comment is not relevant for p/q, only r can drive the system clock.

Reply:
Sounds good. I will update.

Reply:
Updated the comments.